src/utils.cr

```diff
@@ -20,6 +22,15 @@ macro fmt_route(route = "", namespace = false)
        .gsub(/::/, "/") }} {% if namespace %} + "/:namespace" {% end %} + ( {{route}} || "")
 end
 
+# Pull the context from the context's query parameter or use the default
+# provided by the CLI.
+#
+# This macro only works inside of a route block with the variable "context"
+# available
+macro prefix
+  context.params.query["prefix"]? || DPPM.default_prefix
```

Comment:
A macro for this?!  They shouldn't be abused (slower compile time, harder to debug). Use a method taking a context as argument.

Reply:
It can also be moved out of utils, for example in `Actions` or in `DppmRestApi`